### DISCUSSION TITLE
Вывод списка в виде ссылок в поле

### DISCUSSION THREAD

**Discussion by Alexmeb** [totumonline/totum-ru-issues-and-discussions#206] [Как это сделать?] [answered]
В поле номер заказа передается список номеров. 
Возможно ли сделать каждый номер ссылкой на свой источник, чтобы отобрежеие было как на этом скриншоте? 
![Screenshot_34](https://user-images.githubusercontent.com/109726405/187076867-18fb3b15-d804-4403-8447-86ec628ec87e.png)

**Comment by erpic-ru** [comment, marked as answer]
Если бы заказ был один, то без проблем. Насколько я знаю, как нарисовано реализовать не получится.
Рекомендую сделать на это поле linktotable. Либо, как вариант, ссылки можно вывести в панель через panelHtml и listReplace

**Reply by Alexmeb** [reply]
Ну в правую кнопку понятно... хотелось именно в строчную часть)

**Reply by ttm-alexeypolunin** [reply]
В поле никак. В панель по правой кнопке. Можно не только через panelHtml, но и через параметр preview для множественного селекта.
